docs/source/yelpsoa_configs.rst

```diff
@@ -111,6 +111,8 @@ instance MAY have:
     Parsing the Marathon config file will fail if both args and cmd are
     specified [#note]_.
 
+  .. _env:
```

Comment:
I couldn't see a way to make that work. But I'll keep looking (I would love that functionality as well).